### PR TITLE
feat(flagr): Team slug replacing internal teams

### PR DESCRIPTION
**Pull request by wmak**
- The internal team property is being deprecated
- Adding some responses to questions since flagr has been out
- Updating screenshots so that they're easier to read directly on the
  page
- Linking to the screenshots to increase discoverability

### CODE REVIEW

File: src/docs/feature-flags.mdx [hunk 1]
```diff
@@ -235,22 +247,22 @@ segments is that each segment is **OR**ed together, and each constraint is
...
-| `project_id`                    | `int`        | Only passed for projects    |
-| `user_id`                       | `int`        |                             |
-| `user_domain`                   | `string`     |                             |
-| `user_is-superuser`             | `bool`       |                             |
-| `user_is-staff`                 | `bool`       |                             |
-| `internal_teams`                | `[string]`   | Only passed for Sentry      |
+| Property                        | Value Types  | Notes/Example config               |
+| :-----------------------------  | :----------- | :--------------------------------- |
+| `organization_id`               | `int`        |                                    |
+| `organization_slug`             | `string`     | [Example](#releasing-a-feature-to-a-subset-of-organizations) |
```
Comment (wmak) [inline src/docs/feature-flags.mdx:253]: ugh, not sure how to format the `.mdx` to make this less ugly :( now all the `|` are all over the place

Comment (markstory) [reply src/docs/feature-flags.mdx:253]: You could align the cell boundaries with the longest cell?

File: src/docs/feature-flags.mdx [hunk 2]
```diff
@@ -298,11 +310,14 @@ double quotes `"` for each organization.
 
 ![img/flagrOrganizationSubset.png](img/flagrOrganizationSubset.png)
 
-### Releasing a feature to a specific team in sentry
+### Releasing a feature to a specific team
 
-You can enable a feature for only a specific team within the Sentry organization
-by setting the condition property to `internal_team`, the operator to
+You can enable a feature for only a specific team within an oranization
```
Comment (markstory) [inline src/docs/feature-flags.mdx:315]:
```suggestion
You can enable a feature for only a specific team within an organization
```

**Review by markstory — APPROVED** [review]

File: src/docs/feature-flags.mdx (hunk 1 above)
Comment (wmak) [reply src/docs/feature-flags.mdx:253]: 😬 yeah but it gets really long, but I guess that's better than this. 👍

**Pull request merged**